TensorflowInf/requirements.txt

```diff
@@ -1,3 +1,4 @@
-tensorflow==1.15
+tensorflow==1.14
 Pillow
 scipy==1.1.0
```

Comment:
So we no longer need scipy 1.1.0. This version was specified before because we were using scipy.misc(), which is deprecated. We replaced scipy.misc() with PIL.Image.resize(resample = PIL.Image.BILINEAR). The only scipy function we now use is scipy.io.loadmat(), which is up to date in the latest scipy version.